mapper/javabean/src/main/java/org/hibernate/search/mapper/javabean/model/impl/JavaBeanTypeModel.java

```diff
@@ -28,7 +28,7 @@
 import org.hibernate.search.mapper.pojo.util.spi.JavaClassOrdering;
 import org.hibernate.search.util.impl.common.LoggerFactory;
 
-class JavaBeanTypeModel<T> implements PojoRawTypeModel<T> {
+public class JavaBeanTypeModel<T> implements PojoRawTypeModel<T> {
```

Comment:
Making this class public looks unnecessary?

Reply:
You're right. I'll remove them.